tools/chpl-language-server/src/chpl-language-server.py

```diff
@@ -199,6 +199,15 @@ def find(self, pos: Position) -> Optional[EltT]:
             return None
         return self.elts[idx]
 
+    def range(self, rng: Range) -> List[EltT]:
+        start = bisect_right(
+            self.elts, rng.start, key=lambda x: self.get_range(x).start
+        )
+        end = bisect_left(
+            self.elts, rng.end, key=lambda x: self.get_range(x).start
+        )
+        return self.elts[start:end]
+
```

Comment:
This is needed to do the param inlays right? Since thats not in this PR and this PR is already big enough, I would prefer this and the bisect_compat changes be removed from this PR

Reply:
Factored into https://github.com/chapel-lang/chapel/pull/24293